bump upstream version to 0.1.4

jonworek opened the pull request:
Trying to run biscuit on MacOS monterry, you get the following error:

```
fatal error: runtime: bsdthread_register error

runtime stack:
runtime.throw(0x5060e9, 0x21)
	/usr/local/go/src/runtime/panic.go:566 +0x95 fp=0x7ff7bfeff320 sp=0x7ff7bfeff300
runtime.goenvs()
	/usr/local/go/src/runtime/os_darwin.go:88 +0xa0 fp=0x7ff7bfeff350 sp=0x7ff7bfeff320
runtime.schedinit()
	/usr/local/go/src/runtime/proc.go:450 +0x9c fp=0x7ff7bfeff390 sp=0x7ff7bfeff350
runtime.rt0_go(0x7ff7bfeff3c8, 0x1, 0x7ff7bfeff3c8, 0x0, 0x1000, 0x1, 0x7ff7bfeff600, 0x0, 0x7ff7bfeff658, 0x7ff7bfeff66c, ...)
	/usr/local/go/src/runtime/asm_amd64.s:145 +0x14f fp=0x7ff7bfeff398 sp=0x7ff7bfeff390
```

@JonathanWThom in an internal discussion board proved that compiling from the latest source for https://github.com/dcoker/biscuit and replacing the version of the `_biscuit` executable included with the gem would resolve the issue.

My theory is that the latest published release of biscuit should also resolve the error, without manual intervention on the part of someone installing the gem.

## Test Results

Latest 0.1.4 release will resolve the error

<img width="921" alt="image" src="https://user-images.githubusercontent.com/3124548/168136592-39531390-358a-4b90-99eb-8abafe7257ea.png">

tested on

<img width="579" alt="image" src="https://user-images.githubusercontent.com/3124548/168136881-720cf13d-05b5-4a55-b4cf-ac5fa868423f.png">

Commit:
bump upstream version to 0.1.4

With any luck, this will resolve 
`fatal error: runtime: bsdthread_register error`
on latest macos

Commit:
Bump version

frankywahl commented on Rakefile:
```diff
@@ -1,7 +1,7 @@
 require 'open-uri'
 require 'bundler/gem_tasks'
 
-UPSTREAM_VERSION = '0.1.3'
```
You'll also need to change lines 25 - 30 to get this to work. 
The error is most likely linked to the M1 processors on Mac than the OS.

jonworek replied:
Thanks for the tip, @frankywahl I'll check it out. I experienced this error however on an Intel chip. We've had a few folks suspecting the M1 though so I can have them run through this on their machines and see what we get.

Nice to see you here btw. Hope things are well 😄

jonworek replied:
I haven't personally tested M1's yet, but I think what I have will work. If it doesn't I'll submit a follow up release to fix it.

@SP-UT since you have an M1, mind giving this a try?

Commit:
upgrades File.exists to .exist

I believe the old method has been deprecated in ruby
for some time. This might involve needing to drop older
ruby version support.

Commit:
Adds logging

Commit:
Updates to current release url format

Commit:
Adds a few changelog entries

Commit:
Corrects file extension and adds additional logging

jonworek commented:
We tested this yesterday on a few folks machines in a few different scenarios, and put/get operations both seem to work. This looks to be clear to merge.
